diff --git a/meson.build b/meson.build
index a5a31a80..3afde1cc 100644
--- a/meson.build
+++ b/meson.build
@@ -28,12 +28,7 @@ endif
 # https://mesonbuild.com/Python-module.html
 # Here we differentiate from the python used by meson, py3_command, and that python target, py3_target. This is useful
 # when cross compiling like on conda-forge
-py_mod = import('python')
-if get_option('python_target') != ''
-  py3 = py_mod.find_installation(get_option('python_target'))
-else
-  py3 = py_mod.find_installation('python')
-endif
+py3 = import('python').find_installation(pure: false)
 py3_dep = py3.dependency()
 
 message(py3.path())
diff --git a/meson_options.txt b/meson_options.txt
index a768c27b..3b93898c 100644
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -1,9 +1,9 @@
 option('incdir_numpy', type: 'string', value: '',
     description: 'Include directory for numpy. If left empty Meson will try to find it on its own.')
 
-option('python_target', type: 'string', value: '',
-    description: '''Target python path. This is used in the case that the Python installation that PyOptSparse is intended
-    to be built for is different than the Python installation that is used to run Meson. For example, Meson may be installed
-    on the user's system which is run using the system Python installation, but the user may want build PyOptSparse for
-    a Python installation in a virtual environment. Leave as an empty string to build for Python installation running
-    Meson.''')
+#option('python_target', type: 'string', value: '',
+#    description: '''Target python path. This is used in the case that the Python installation that PyOptSparse is intended
+#    to be built for is different than the Python installation that is used to run Meson. For example, Meson may be installed
+#    on the user's system which is run using the system Python installation, but the user may want build PyOptSparse for
+#    a Python installation in a virtual environment. Leave as an empty string to build for Python installation running
+#    Meson.''')
diff --git a/wisdem/meson.build b/wisdem/meson.build
index c4999da6..713d3ac9 100644
--- a/wisdem/meson.build
+++ b/wisdem/meson.build
@@ -1,13 +1,13 @@
 # NumPy include directory - needed in all submodules
-incdir_numpy = get_option('incdir_numpy')
-if incdir_numpy == ''
-    incdir_numpy = run_command(py3,
-      [
-        '-c',
-        'import os; os.chdir(".."); import numpy; print(numpy.get_include())'
-      ],
-      check: true
-    ).stdout().strip()
+incdir_numpy = meson.get_external_property('numpy-include-dir', 'not-given')
+if incdir_numpy == 'not-given'
+  incdir_numpy = run_command(py3,
+                             [
+                               '-c',
+                               'import os; os.chdir(".."); import numpy; print(numpy.get_include())'
+                             ],
+                             check: true
+                            ).stdout().strip()
 endif
 # this creates a raw string which is useful for Windows use of '\' for paths
 incdir_numpy = '''@0@'''.format(incdir_numpy)
